File: Array/1295-Find-Numbers-with-Even-Number-of-Digits/1295-Find-Numbers-with-Even-Number-of-Digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for(int num : nums) {
            if(isEvenDigit(num)) {
                count++;
            }
        }
        return count;
    }
    
    public boolean isEvenDigit(int num) {
        int digit = (int)(Math.floor(Math.log10(num)) + 1);
        return (digit % 2) == 0;
    }
}

File: Array/1295-Find-Numbers-with-Even-Number-of-Digits/1295-Find-Numbers-with-Even-Number-of-Digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
        let count = 0;
        nums.map(num => {
            if(isEvenDigit(num)) {
                count++;
            }
        })
        return count;
  
};
    function isEvenDigit(num) {
        let digit = (Math.floor(Math.log10(num)) + 1);
        return (digit % 2) == 0;
    } 

